runtime/frequency/src/lib.rs

```diff
@@ -633,16 +630,46 @@ impl pallet_aura::Config for Runtime {
 impl pallet_collator_selection::Config for Runtime {
 	type Event = Event;
 	type Currency = Balances;
-	type UpdateOrigin = CollatorSelectionUpdateOrigin;
-	type PotId = CollatorPotId;
-	type MaxCandidates = CollatorMaxCandidates;
-	type MinCandidates = CollatorMinCandidates;
-	type MaxInvulnerables = ConstU32<10>;
+
+	// Origin that can dictate updating parameters of this pallet.
+	// Currently only root or a 3/5ths council vote.
+	type UpdateOrigin = EitherOfDiverse<
+		EnsureRoot<AccountId>,
+		pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 3, 5>,
+	>;
+
+	// Account Identifier from which the internal Pot is generated.
+	// Set to something that NEVER gets a balance i.e. No block rewards.
+	type PotId = NeverDepositIntoId;
+
+	// Maximum number of candidates that we should have. This is enforced in code.
+	//
+	// This does not take into account the invulnerables.
+	type MaxCandidates = ConstU32<0>;
+
+	// Minimum number of candidates that we should have. This is used for disaster recovery.
+	//
+	// This does not take into account the invulnerables.
+	type MinCandidates = ConstU32<0>;
+
+	// Maximum number of invulnerables. This is enforced in code.
+	type MaxInvulnerables = ConstU32<16>;
+
+	// Will be kicked if block is not produced in threshold.
 	// should be a multiple of session or things will get inconsistent
-	type KickThreshold = SessionPeriod;
+	type KickThreshold = ConstU32<{ 6 * HOURS }>;
```

Comment:
I removed the type alias references from `common/src/constants.rs` in favor of specifying each `ConstU32` struct with its value. My thinking is that it is easier to glance at the pallet config and immediately know the values of each type. In the spirit of separate individually configurable runtimes, this seems to help?

Reply:
We might have separate sessionperiods ? 

Not sure the SessionPeriod is set using macro prod_or_local_env

Reply:
After some discussions with Puneet, I will kick back the values to runtime common constants and allow them to be independently configurable with a slightly modified `prod_or_local_env` macro.